app/views/registrations/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Create Account</h2>
+
+<%= devise_error_messages! %>
+
+<%# View for Devise method inherited by Registrations controller. %>
+<%# Uses `resource` for consistency with Devise::Registrations#new %>
+<%= form_for(resource,
+             as: resource_name,
+             url: registrations_path(resource),
+             html: { class: 'pss-admin-form' }) do |f| %>
+
+  <%= f.label :email %>
+  <%= f.email_field :email, autofocus: true %>
+
+  <%= f.label :password %>
+  <%= f.password_field :password, autocomplete: "off" %>
+  <% if @validatable %>
+  <div><em>(<%= @minimum_password_length %> characters minimum)</em></div>
+  <% end %>
+
+  <%= f.label :password_confirmation %>
+  <%= f.password_field :password_confirmation, autocomplete: "off" %>
```

Comment:
I thought I did remove that already, but I could be confused.  Let me check; may have committed it elsewhere.

I intended for there to be no password form on the create page.